src/status_im/ui/components/icons/vector_icons.cljs

```diff
@@ -202,6 +202,7 @@
      ;; Tiny icons
      :tiny-icons/tiny-arrow-down   (components.svg/slurp-svg "./resources/icons/tiny/tiny_arrow_down.svg")
      :tiny-icons/tiny-check        (components.svg/slurp-svg "./resources/icons/tiny/tiny_check.svg")
+     :tiny-icons/tiny-pending      (components.svg/slurp-svg "./resources/icons/tiny/tiny_pending.svg")
```

Comment:
please alphabetical order and also import for desktop as well just in case

Reply:
Done.